### PR TITLE
Add configuration for forceAccess. Add logging.

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java [hunk 1]
```diff
@@ -89,14 +92,19 @@ public Object deserialize( JsonParser p, DeserializationContext ctxt ) throws IO
         StreamSupport.stream( beanDescription.getClassInfo().memberMethods().spliterator(), true ).forEach( m -> {
             if ( m.hasAnnotation( JsonPostDeserialize.class ) )
             {
-                logger.debug( "Invoking method {} ", m.getName() );
+                logger.info( "Invoking method {} on {} ", m.getName(), beanDescription.getBeanClass() );
```
Comment (goldmann) [inline src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: I would probably leave this as debug.

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java [hunk 2]
```diff
@@ -56,7 +59,7 @@ public static Module getSimpleModule()
                 if ( StreamSupport.stream( beanDescription.getClassInfo().memberMethods().spliterator(), true).
                                 anyMatch( m -> m.hasAnnotation( JsonPostDeserialize.class ) ) )
                 {
-                    logger.debug( "BeanDescription {} ", beanDescription.getClassInfo());
+                    logger.info( "BeanDescription {} ", beanDescription.getClassInfo());
```
Comment (goldmann) [inline src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: Same here, debug seems to be fine.

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java [hunk 3]
```diff
@@ -46,6 +46,9 @@
     public static Module getSimpleModule()
     {
         SimpleModule module = new SimpleModule();
+        System.out.println ( "### Registering simple module (new snapshot)");
+        logger.info ( "### Registering simple module with logger.info");
+        logger.debug ( "### Registering simple module with logger.debug");
```
Comment (goldmann) [inline src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: :grinning:

Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: After your presentation lets try clearing the grape cache and seeing what does appear. I think my preference is for the `logger.info` for this case.

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java (hunk 2 above)
Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: Agreed

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java (hunk 1 above)
Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: Not sure .... using as a java library I would agree.When its being run inside jenkins I thinks it useful.

Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: I wonder if we supply a default `logging.properties` configured to debug (I think Jenkins uses JUL) and it works just for jenkins ?

Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: Reverted to debug ; will deal with Jenkins separately.

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java (hunk 3 above)
Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: Reverted to debug ; will deal with Jenkins separately.

File: src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java (hunk 2 above)
Comment (rnc) [reply src/main/java/org/goots/jackson/deserializer/JacksonPostHookDeserializer.java]: Reverted to debug ; will deal with Jenkins separately.

**Pull request merged**